components/create_team/components/display_name/index.js

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2018-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {connect} from 'react-redux';
+
+import TeamSignupDisplayNamePage from './display_name';
+
+export default connect(null)(TeamSignupDisplayNamePage);
```

Comment:
No action needed, just a heads-up for the future:

If there's nothing happening here you can actually leave this file out and instead of having the directory for the component, you can just have the component file